allennlp/training/metrics/mean_absolute_error.py

```diff
@@ -66,9 +66,10 @@ def get_metric(self, reset: bool = False):
         mean_absolute_error = self._absolute_error / self._total_count
         if reset:
             self.reset()
+        assert isinstance(mean_absolute_error, float)
```

Comment:
Since we have this assertion covered in the tests, let's remove it here.

Reply:
Ah, sorry! That was a sanity check for me. I'll remove it.